nih-grants/.htaccess

```diff
@@ -0,0 +1,26 @@
+# Example
+#
+# https://w3id.org/nih-grants/NOT-LM-23-001 redirects to https://grants.nih.gov/grants/guide/notice-files/NOT-LM-23-001.html
+# https://w3id.org/nih-grants/PA-20-184 redirects to https://grants.nih.gov/grants/guide/pa-files/PA-20-184.html
+# https://w3id.org/nih-grants/PAR-23-034 redirects to https://grants.nih.gov/grants/guide/pa-files/PAR-23-034.html
+# https://w3id.org/nih-grants/PAS-25-236 redirects to https://grants.nih.gov/grants/guide/pa-files/PAS-25-236.html
+# https://w3id.org/nih-grants/RFA-OD-19-029 redirects to https://grants.nih.gov/grants/guide/rfa-files/RFA-OD-19-029.html
+#
+# ## Contact
+# This space is administered by:
+#
+# Clair Kronk
+# Clair.Kronk@mountsinai.org
+# GitHub username: Superraptor
+
+# Examples tested using:
+# https://htaccess.madewithlove.com/
+# (27 January 2025)
+
+# Redirect URLs of the form /nih-grants/$1
+RewriteRule ^((NOT)(-)([A-Z]{2})(-)([0-9]{2})(-)([0-9]{3}))$ https://grants.nih.gov/grants/guide/notice-files/$1.html [R=301,L]
+RewriteRule ^((PA)(-)([0-9]{2})(-)([0-9]{3}))$ https://grants.nih.gov/grants/guide/pa-files/$1.html [R=301,L]
+RewriteRule ^((PAR)(-)([0-9]{2})(-)([0-9]{3}))$ https://grants.nih.gov/grants/guide/pa-files/$1.html [R=301,L]
+RewriteRule ^((PAS)(-)([0-9]{2})(-)([0-9]{3}))$ https://grants.nih.gov/grants/guide/pa-files/$1.html [R=301,L]
+RewriteRule ^((RFA)(-)([A-Z]{2})(-)([0-9]{2})(-)([0-9]{3}))$ https://grants.nih.gov/grants/guide/rfa-files/$1.html [R=301,L]
+RewriteRule ^((RFS)(-)([A-Z]{2})(-)([0-9]{2})(-)([0-9]{3}))$ https://grants.nih.gov/grants/guide/???/$1.html [R=301,L]
```

Comment:
Did you want to fix the `???` before merging?

Reply:
Apologies-- although I saw RFS/RFP mentioned as a prefix here (https://nexus.od.nih.gov/all/2010/07/01/2-what-is-the-difference-between-a-pa-par-pas-rfa-and-rfp-2/), investigating the NIH funding opportunity types under the advanced search here (https://grants.nih.gov/funding/nih-guide-for-grants-and-contracts), it doesn't list RFP or RFS and I can't find examples anywhere else. If I do find them in the future I'll add them, but as of now I just removed the line and updated.